src/Network/Network/help/Add-AzLoadBalancerProbeConfig.md

```diff
@@ -24,13 +24,22 @@ The **Add-AzLoadBalancerProbeConfig** cmdlet adds a probe configuration to an Az
 
 ## EXAMPLES
 
-### Example 1 Add a probe configuration to a load balancer
-```
+### Example 1: Add a probe configuration to a load balancer
+```powershell
 PS C:\>Get-AzLoadBalancer -Name "myLb" -ResourceGroupName "myRg" | Add-AzLoadBalancerProbeConfig -Name "probeName" -RequestPath healthcheck2.aspx -Protocol http -Port 81 -IntervalInSeconds 16 -ProbeCount 3 | Set-AzLoadBalancer
 ```
 
 This command gets the load balancer named myLb, adds the specified probe configuration to it, and then uses the **Set-AzLoadBalancer** cmdlet to update the load balancer.
 
+### Example 2
+
+Adds a probe configuration to a load balancer. (autogenerated)
+
+<!-- Aladdin Generated Example -->
+```powershell
+Add-AzLoadBalancerProbeConfig -IntervalInSeconds 15 -LoadBalancer <PSLoadBalancer> -Name 'NewProbe' -Port 80 -ProbeCount 2 -Protocol 'https'
```

Comment:
I find it odd that port 80 would be set to HTTPS. By default, port 80 is HTTP and port 443 is HTTPS. A different port might be used for testing, but I've never seen anyone run SSL across port 80.